src/main/java/bio/terra/service/resourcemanagement/google/GoogleResourceService.java

```diff
@@ -405,6 +409,16 @@ public void grantPoliciesBqJobUser(String dataProject, Collection<String> policy
         enableIamPermissions(policyMap, dataProject);
     }
 
+    public void revokePoliciesBqJobUser(String dataProject, Collection<String> policyEmails)
+        throws InterruptedException {
+
+        Map<String, List<String>> policyMap = new HashMap<>();
+        List<String> emails = new ArrayList<>(CollectionUtils.collect(policyEmails, e -> "group:" + e));
```

Comment:
If this is the same as
```suggestion
        List<String> emails = policyEmails.stream().map(e -> "group:" + e).collect(Collectors.toList());
```
then I don't see the improvement of using this form.

Reply:
ok fine :)